### ISSUE TITLE
Accessibility issue: cannot zoom with ctrl+plus

### ISSUE THREAD

**Issue by snotskie** [marp-team/marp-core#393]
The svg containers for Marp slides are set to 100% viewport width and height.

When one of my collaborators, who is legally blind with low vision, tries to zoom in on her desktop computer using ctrl+plus as she normally does for websites, the Marp slides do not change size, because they rescale to the new viewport size. In effect, Marp disables zoom.
